aegis/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>aegis</name>
+  <version>0.0.1</version>
```

Comment:
Set the initial version of the all packages to the `0.1.0` to mark out our previous development.

```suggestion
  <version>0.1.0</version>
```